docs/release/release_0_4_14.md

```diff
@@ -0,0 +1,68 @@
+# napari 0.4.14
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
```

Comment:
```suggestion
This is a small release containing some new features and fixing a few issues that have come up since 0.4.13 was released.
```

I think we should be consistent about whether version numbers are in backticks. Version 0.4.14 is not above and I think backticks are mostly reserved for code, so I suggest removing them here.